src/Components/AsideNav.js

```diff
@@ -32,10 +32,20 @@ class AsideNav extends Component {
     });
   };
 
+  handleBg = () => {
+    this.setState({
+      close: false,
+    });
+  };
+
   render() {
     return (
       <div className="AsideNav">
         <Nav open={this.openAside} />
+        <div
+          className={this.state.close ? "bg" : "no"}
+          onClick={this.handleBg}
+        ></div>
```

Comment:
자식 요소 없으면 self closing!
```suggestion
        <div
          className={this.state.close ? "bg" : "no"}
          onClick={this.handleBg}
        />
```